recipe/build.sh

```diff
@@ -66,3 +66,8 @@ find . -name "*activate*.sh" -exec sed -i.bak "s|@_CONDA_PYTHON_SYSCONFIGDATA_NA
 find . -name "*activate*.sh" -exec sed -i.bak "s|@CONDA_BUILD_CROSS_COMPILATION@|${CONDA_BUILD_CROSS_COMPILATION}|g"                "{}" \;
 
 find . -name "*activate*.sh.bak" -exec rm "{}" \;
+
+# Check if (de-)activate scripts can be used in non-Bash shells (ignoring the commonly supported "local" keyword.)
+find . -name "*activate*.sh" -exec shellcheck --severity=info --format=gcc {} \; \
+    | grep -vF "In POSIX sh, 'local' is undefined. [SC" \
```

Comment:
Probably better to pass `-e SC3043` to `shellcheck` instead of this grep.

Reply:
Nice! I googled for this and couldn't find it. I should have checked the man page!

Reply:
Unfortunately, it appears the error code has changed since several months ago! 

It was SC2039 but now it comes out as SC3043!

:/

Reply:
Yeah, the single generic error was split into multiple specific errors.  I think overall this is a good thing, as it allows finer-grained ignoring.